### DISCUSSION TITLE
How to build without need to run build on host machine?

### DISCUSSION THREAD

**Discussion by eakarpov** [vercel/next.js#37286] [Help]
Hi!

Is it possible to build application with all dependencies needed? So that I just copied the output folder on the host and run a script with Node.js on it to start an application? Why should I need to call build on host and have node_modules installed and package.json?

**Comment by arielvieira** [comment]
@eakarpov I believe this is what you want  https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files-experimental
